modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -967,6 +967,12 @@ class CV_EXPORTS_W VideoCapture
             CV_OUT std::vector<int>& readyIndex,
             int64 timeoutNs = 0);
 
+    CV_WRAP static
+    std::vector<int> waitAny(
```

Comment:
This overload is unnecessary.

```cxx
CV_WRAP static waitAny(const std::vector<VideoCapture>& streams,
                       CV_OUT std::vector<int>& readyIndex,
                       int64_t timeoutNs = 0);
``` 
should work if you add missing conversion function for `int64_t`
```cxx
template<> bool pyopencv_to(PyObject* obj, int64_t& value, const ArgInfo& info);
```

and you should add an appropriate test for it to `modules/python/test/test_misc.py`

Reply:
Overload with `vector<int>` return us useful (see #21826 `waitAny()` signature proposal). But I can use old signature, if needed.

Anyway, I'll add missing conversion for `int64_t` and test for it.

Reply:
The signature will be:
```python
ret_code, indices = cv2.VideoCapture.waitAny(captures)
if ret_code:
    ...
```

Not the best looking API, but it is consistent with all functions having `CV_OUT` arguments.

Reply:
I have an isuue with int64 test. 

I added `dumpInt64` function to `modules/core/include/opencv2/core/bindings_utils.hpp`

```cpp
CV_WRAP static inline
String dumpInt64(int64 argument)
{
    return cv::format("Int: %ld", argument);
}
```

But on some systems int64 is long and on other systems int64 is long long. 

How can I prorperly handle it?

Reply:
Same thing with `size_t`. It was handled with `std::ostringstream`